resources/i18n/fr.json

```diff
@@ -68,17 +69,15 @@
 	"download-directory-setting": "Dossier de téléchargement :",
 	"reset": "Réinitialiser",
 	"browse": "Parcourir",
-	"kiwix-desktop-description": "Kiwix vous permet d'avoir tout Wikipedia à portée de main où que vous alliez! Sur un bateau, au milieu de nulle part ou en prison, Kiwix vous donne accès à l'ensemble du savoir humain. Vous n'avez pas besoin d'internet, tout est conservé dans votre ordinateur.",
-	"learn-more-about-kiwix": "En apprendre plus sur Kiwix",
-	"release": "Release",
-	"source-code": "Ce logiciel est publié selon les conditions de la licence GNU General Public License version 3. Voir le code source sur",
-	"version": "Version",
-	"build-date": "Date de compilation",
-	"reporting-problem": "Rapporter un problème",
-	"report-issue-1/3": "Pour ouvrir un rapport de bogue, visitez",
-	"report-issue-2/3": "et ouvrez un ticket en décrivant votre problème.",
-	"report-issue-3/3": "Veuillez mentionner la version et la date de compilation dans le rapport.",
-	"libraries": "Bibliothèques",
+	"about-kiwix-desktop-title": "Bureau Kiwix",
+	"about-kiwix-desktop-description": "Kiwix vous permet d'avoir tout Wikipedia à portée de main où que vous alliez! Sur un bateau, au milieu de nulle part ou en prison, Kiwix vous donne accès à l'ensemble du savoir humain. Vous n'avez pas besoin d'internet, tout est conservé dans votre ordinateur.",
+	"about-learn-more-about-kiwix": "En apprendre plus sur Kiwix",
+	"about-release-title": "Release",
+	"about-version": "Version : {{VERSION}}",
+	"about-reporting-problem-title": "Rapporter un problème",
+	"about-report-issue": "Pour déclarer une faute, allez sur <a href='{{TRACKER_URL}}'>Suivi des problèmes du bureau Kiwix</a> et ouvrez un ticket en décrivant votre problème.",
```

Comment:
"Faute" seems a bit wrong here. I would translate it by "bogue" (or simply keep keep the previous translation : "Pour ouvrir un rappor de bogue" instead of "Pour déclarer une faute".